documentation/source/user_section/command-line/deepseg/lesion_ms.rst

```diff
@@ -0,0 +1,11 @@
+
+                
+lesion_ms
+=========
+                
+.. argparse::
+   :ref: spinalcordtoolbox.scripts.sct_deepseg.lesion_ms
+   :prog: sct_deepseg lesion_ms
+   :markdownhelp:
+   :noepilog:
```

Comment:
Is there a reason for this `:noepilog:` thing? It looks like only 4 of the models have it. I don't see an epilog when I run `sct_deepseg lesion_ms -h`, for what it's worth.

Reply:
Good catch!!

We use `epilog=` to display the list of tasks when running the top-level `sct_deepseg -h`:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/d1c1cb248bfd153d9da454c21e49b72ec76152e8/spinalcordtoolbox/scripts/sct_deepseg.py#L39

So `:noepilog:` was used to avoid showing the list of tasks on every single individualized `sct_deepseg` page (because the model gallery basically serves that role now).

However, with the switch to subparsers, we no longer run `sct_deepseg -h` on every individual page, so there's no epilog to hide! It looks like I missed some instances of `:noepilog:`. Either that, or a merge from `master` brought in some new models, and I forgot to clean up the incoming pages.